docs/usage/config-presets.md

```diff
@@ -151,7 +151,7 @@ To host your preset config on GitLab:
 ## Gitea-hosted Presets
 
 It is also possible to host your preset config using just a regular Gitea repository and without needing to publish it to npmjs.
-In such cases Renovate will simply look for a `renovate.json` file in the default branch, (for now only the _master_ branch is supported).
+In such cases Renovate will simply look for a `renovate.json` file in the base branch.
```

Comment:
```suggestion
In such cases Renovate will simply look for a `renovate.json` file in the default branch.
```

@zharinov can you also check here re: only master being supported, or only default being supported?

Reply:
https://try.gitea.io/api/swagger#/repository/repoGetContents
We're passing `ref` only if supported, so any default branch will be used.